delfin/drivers/dell_emc/unity/unity.py

```diff
@@ -81,7 +75,7 @@ def get_storage(self, context):
                 'location': '',
                 'subscribed_capacity': int(subs),
                 'total_capacity': int(total),
-                'raw_capacity': int(raw),
+                'raw_capacity': int(total),
```

Comment:
Both total capacity and raw_capacity is mapped to same value now?
 total_capacity should be total usable capacity and raw capacity is the sum of all raw disks !